### DISCUSSION TITLE
Question: Can datree replace https://github.com/zegl/kube-score ?

### DISCUSSION THREAD

**Discussion by noorul** [datreeio/datree#841] [General]
We are using kube-score in our CI system to validate certain policies. Can datree be used as replacement?

**Comment by eyarz** [comment]
Yes, it can. 
The majority of [kube-score checks](https://github.com/zegl/kube-score/blob/master/README_CHECKS.md) already exist as [built-in rules](https://hub.datree.io/built-in-rules/rules/#Strict) at Datree. 
Missing checks can be added by creating [custom rules](https://hub.datree.io/custom-rules/basic-examples).

**Comment by noorul** [comment]
Looks like I need to publish the custom rules to some centralized location. Is there a way to avoid that?

**Reply by eyarz** [reply]
It's not mandatory, just recommended. 
If you don't want to publish the policies to your account, you can still pass them to Datree every time you run a policy check:
1. [enable](https://hub.datree.io/setup/policy-as-code#1-enable-policy-as-code-pac-mode) policy-as-code mode
2. create a [policy-as-code file](https://hub.datree.io/setup/policy-as-code#policiesyaml)
3. use the [flag](https://hub.datree.io/setup/cli-arguments) ` --policy-config` when running datree

**Comment by noorul** [comment]
cool, one final question! Can the report dashboard be hosted locally? Or is there a way to dump the result as HTML?

**Reply by eyarz** [reply]
The dashboard can't be hosted locally.
But if your use case is dumping the result into a file, you can open a feature request (on this project), and we will prioritize it :)

**Reply by noorul** [reply]
Great! https://github.com/datreeio/datree/issues/846
